ansible/vars/production/production_public.yml

```diff
@@ -35,7 +35,6 @@ primary_ssl_env: "production"
 dev_group: 'dimagidev'
 dev_users:
   present:
-    - ansible
```

Comment:
They are all the same so we could do it. It would mean you'd have to add an extra command line arg with that vars file.